requirements/deploy.txt

```diff
@@ -149,7 +149,7 @@ django-elasticsearch-dsl==8.0
     # via -r requirements/pip.txt
 django-extensions==3.2.3
     # via -r requirements/pip.txt
-django-filter==24.3
+django-filter==25.1
```

Comment:
Nothing super relevant to us https://github.com/carltongibson/django-filter/blob/main/CHANGES.rst#version-251-2025-02-14